labelbox_dev/data_row.py

```diff
@@ -179,9 +184,59 @@ def get_by_global_keys(
                 raise LabelboxError(
                     f"Failed to retrieve task information for `data_row.get_by_global_keys()` async operation"
                 )
-            return BulkGetDataRowsTask.get_by_id(task_id)
+            return GetDataRowsTask.get_by_id(task_id)
 
         return IdentifierPaginator(f"{DATA_ROW_RESOURCE}",
                                    DataRow,
                                    global_keys,
                                    identifiers_key='global_keys')
+
+    @staticmethod
+    def update_many(data_rows: List[UpdateDataRowsWithIdType],
```

Comment:
any tests?

Reply:
I have some tickets to add test coverage. Since this is dependent on intelligence repo changes, I will address tests in a separate ticket

Reply:
Created https://labelbox.atlassian.net/browse/AL-4970 for writing tests for DataRow and Dataset modules